### DISCUSSION TITLE
[Question] Wrapper of torchrl

### DISCUSSION THREAD

**Discussion by Omikaa** [isaac-sim/IsaacLab#499] [Q&A]
### Question

I want to use torchrl with isaac lab, but it is too difficult for me to figure out the Isaac sim Env and torchrl parallelenv?
Is there any plan to supply a wrapper for torchrl?

**Comment by Dhoeller19** [comment]
Hi @Omikaa,
Indeed this is on our roadmap.

**Comment by Omikaa** [comment]
Glad to head that! 
Thanks for your reply. Hoping that it will be done quickly.
